manifests/overlays/prod/secrets/clusters/api.ocp4.idh.dev.datahub.redhat.com.enc.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: cluster-api.ocp4.idh.dev.datahub.redhat.com
+    labels:
+        argocd.argoproj.io/secret-type: cluster
+    annotations:
+        managed-by: argocd.argoproj.io
+type: Opaque
+stringData:
+    name: api.ocp4.idh.dev.datahub.redhat.com
+    config:
+        bearerToken: ENC[AES256_GCM,data:XpQTT5tPOggs1PBr3WRU2zh68UsRIfV27AAVfqtppZXoE4ql/iztZOXpkPia9B5aHxRU72MHMoSk/ZuNYZCZYdZCsp8apdiYbzCTWsGZATtJuPu8ACejB4PB/h+qOzmwy+I3GuYqSRj9EMAUTjxKVi1KPyEJRYagAp2J2Jm5GQ0kzI+lXaqR39idFb6gja1XIx7lDp5n7PD/HMYmULc/FpHxAnFfcFElwOchGPGET/OsbUacXXrrmSxsgndTzZlpiIX7WPbtp31Nd5/2ucYRSddRe7GIwKqUG6xDPpez+PfsbYHaiF6pDT1ViwsnX44eE4VHRVB7MaeeaK4NF+cXUrCls66xtz+EBtulNX0GnuPh0yw5ajxfWbg6LPUFN6L0hKW/yWXL9qScB+/n8U8XI9TiUK7JXwjEjS5nsP9gXvl+fkQ03iVX4FghnyQYbl+/RytfUV5N1yAgL8kSxJq91xaIUrciEEMOQCWm/f+XRGX7ro+wdMxlf/OMT5G25Mw7CtzM0UuAhFiEcPXdISwUzGT8Ekf/9tCZt1glRAForNH4Debpmg7v1+5GxSubqVnooCGZ2G2DXO8ByrNdOR5/eOGbq7tjI3KX33Qitx3fs11r88apw1HF5gFbEid/lctWv7WdyE6WPhxI2SeTsaP6Fugk0dag+My2R9622WAKE1XD432IScFRe8onjHPCiIhpLyQXMJD7J6Uu5BJ76qd4/tHA0opNLkcfVtMZCW27As1sbQxjhOlS+wjzpIY+bClh4MtPc7nf/TKZR7mKguklqWDos6jQoUmWmvEQ4VhJc2KDnWUW7SpBzg6ICp6d0Mbk+NPEds31HBelUCUhUmfdm68vJWn0cOOHKSdoSDonVfa9xeuHQOTqr9WmeYbsYDRAU1sj7ANgGHP2Tnq6/U8jBCR7AFUWrmNVacREbpD1WkgJMvwn0BtIKeZqjbALgEG7n9i+CAx2qcqzAUFkdxfAMLXu+LM/zRpfsGuxrcfU3uzhjLinprNUhl+H9EOQQu8lRmqu2h0fpXsIMavMuG5L0Mf8qGRAfzLxE/0JgdDHndOFEs06DM0W0ubo/aPNHeLww/sLsl4KvstkCTqGb1z+n6mFf4PnTl/FAWk/af4Ak3PiEmI1nqcMHU9ttlmB1JzMdmOo1Q0wBaZHP6apsN6TNx713vjix/agg2hgugEHT4atYZuwHdPICJdE1cclIHMJzGJbq+PTiECwGSbnINB0tqQSYpgdAjyGWuYU1Z3wW9UjHw==,iv:bz0fRNxwwJAmTxyPf8/W8+GyxPfMeA+PhgvN9pQwrQo=,tag:l3vZZl8i3DPaDP+9vwYr8w==,type:str]
+        tlsClientConfig:
```

Comment:
looks like you have the jwt token as yaml, it should be json, i.e. of the form: 
```
config: | 
     {"bearerToken": "token", "tlsClientConfig": {"insecure: true}} 
```
what you have: 
```
config: 
   bearerToken: token
   tlsClientConfig: 
      insecure: true
```

Reply:
fixed